subprojects/language-native/src/integTest/groovy/org/gradle/language/cpp/CppToolChainChangesIntegrationTest.groovy

```diff
@@ -129,8 +129,12 @@ class CppToolChainChangesIntegrationTest extends AbstractIntegrationSpec {
         def availableToolChains = AvailableToolChains.toolChains.findAll {
             it.available && !(it instanceof AvailableToolChains.InstalledSwiftc)
         }
+        println("AvailableToolChains: $availableToolChains")
         int numberOfToolChains = availableToolChains.size()
-        Assume.assumeTrue('2 or more tool chains are required for this test', numberOfToolChains >= 2)
+        if (numberOfToolChains < 2) {
```

Comment:
Instead of `Assume`, which is JUnit 4, we should now use `Assumptions`, which is JUnit 5, right?

Reply:
Moreover, we have `spock-junit4` on the classpath, so this shouldn't be a problem I think.

Reply:
See the link in comment: https://github.com/spockframework/spock/issues/1185

This is suggested by Vampire.

Reply:
I read the issue, and that is why I suggest we use the JUnit 5 replacement `Assumptions` or continue to use `Assume`, given that we have `spock-junit4` on the classpath.